assets/CAIP-25/openrpc.json

```diff
@@ -0,0 +1,291 @@
+                  ],
+                  "events": [
+                    "accountsChanged",
+                    "chainChanged"
+                  ],
+                  "accounts": [
+                    "eip155:42161:0x0910e12C68d02B561a34569E1367c9AAb42bd810"
+                  ],
+                  "cosmos": {}
+                },
+                "sessionProperties": {
+                  "expiry": "2022-11-31T17:07:31+00:00"
+                }
+              }
+            }
+          }
+        }
+      ],
+      "errors": [
+        {
+          "code": 5000,
+          "message": "User disapproved requested chains"
+        },
+        {
+          "code": 5001,
+          "message": "User disapproved requested methods"
+        },
+        {
+          "code": 5002,
+          "message": "User disapproved requested events"
+        },
+        {
+          "code": 5100,
+          "message": "Requested chains are not supported"
+        },
+        {
+          "code": 5101,
+          "message": "Requested methods are not supported"
+        },
+        {
+          "code": 5102,
+          "message": "Requested events are not supported"
+        },
+        {
+          "code": 5103,
+          "message": "Scope/chain mismatch"
+        },
+        {
+          "code": 5104,
+          "message": "ChainId defined in two different scopes"
+        },
+        {
+          "code": 5200,
+          "message": "Invalid Session Properties requested"
+        },
+        {
+          "code": 5201,
+          "message": "Session Properties can only be optional"
+        }
+      ]
+    }
+  ],
+  "components": {
+    "schemas": {
+      "Namespace": {
+        "type": "object",
+        "title": "Namespace",
+        "description": "Namespace for a multi-chain connection",
+        "additionalProperties": true,
+        "properties": {
+          "chains": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "pattern": "[-a-z0-9]{3,8}(:[-_a-zA-Z0-9]{1,32})?"
+            }
+          },
+          "methods": {
+            "description": "Methods that the wallet must support in order to be used with this provider.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "events": {
+            "description": "Events that the wallet must support in order to be used with this provider.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "rpcEndpoints": {
+            "description": "JSON-RPC endpoints for this namespace.",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "uri"
+            }
+          },
+          "rpcDocuments": {
+            "type": "array",
+            "description": "OpenRPC documents that describe the RPC endpoints for this namespace.",
```

Comment:
that describe the endpoints or the methods?

Reply:
```suggestion
            "description": "OpenRPC documents that define RPC methods in which to anchor the methods authorized in a CAIP-25 interaction.",
```

Reply:
I'm with Hassan here-- an RPC doc might be useful in justifying (or establishing trust towards) a request to use specific endpoints, but I can imagine usecases for either of these properties to be called in isolation.  For example, a `wallet_` or offchain method might be defined in an RPC doc, or a special endpoint could be requested for non-RPC-method reasons (i.e., use my endpoint because it's faster, more private, etc, but I won't ask you to use any non-standard methods, execution-api is fine here)